source/localizable/index.html.erb

```diff
@@ -1,12 +1,11 @@
 ---
 title:
   en: "LearningSpaces: Focus your team’s learning efforts"
-  nl: "LearningSpaces: Focus your team’s learning efforts"
```

Comment:
As defined in nl.yml 

`default_title: "LearningSpaces: Een centrale plek voor kennis binnen je team"`

Needs this to be added to the frontmatter?